### PR TITLE
completed seller search functionality so seller can search inventory by keyword

### PR DESCRIPTION
**Pull request by aland1013**

### CODE REVIEW

File: src/components/common/searchbar/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function SearchBar({ searchVisible, setData }) {
       <div className="searchOuter">
         <div className="searchBtns"></div>
         <Search
-          defaultValue="Search through your inventory"
+          placeholder="Search through your inventory"
```
Comment (BkAngel201) [inline src/components/common/searchbar/index.js:34]: its anoying to have to take all the text on the field to type yours placeholder was a really good call here

**Review by christian-mack — APPROVED** [review]
Looks good. This is what was changed in our zoom meeting.

File: src/components/sellerPages/inventory/current/index.js [hunk 2]
```diff
@@ -17,18 +17,15 @@ function CurrentInventory({ inventory, fetchProducts, getProductsStatus }) {
...
 
-  const displayedData = useSearch(inventory, 'name', searchData);
-  console.log(inventory);
   return (
     <>
       <NavBar searchVisible={false} setData={setSearchData} />
       <div className="outerContainer">
         <div className="contents">
-          <SearchResults data={displayedData} filter={searchData} />
+          <SearchResults data={inventory} filter={searchData} />
```
Comment (BkAngel201) [inline src/components/sellerPages/inventory/current/index.js:25]: really a headache to figure out what was making the 2nd call here but we got it, nicely done

File: src/components/common/searchbar/index.js (hunk 1 above)
Comment (pjose92) [inline src/components/common/searchbar/index.js:34]: Great job on changing this line on our zoom meeting. Works a lot better.

**Review by BkAngel201 — APPROVED** [review]
everyone is getting feedback and understanding how this code works we are making progress here guys

**Pull request merged**